src/src/AdminBundle/Resources/views/Organizers/organizers_edit.html.twig

```diff
@@ -0,0 +1,40 @@
+<h2>Edit organizer</h2>
+      <hr>
```

Comment:
hmm... preco je to odformatovane doprava? cely tento subor posun o tabulator dolava....